src/cpp/config.cpp

```diff
@@ -0,0 +1,60 @@
+#include <fstream>
+#include <iostream>
+#include <sstream>
+#include <string>
+extern "C" {
+  #include "config.h"
+}
+
+using namespace std;
+
+/**
+ * Parses the provided config file,
+ * and stores the results in the provided structure.
+ */
+void parse_config(const char *filename, Config *config) {
+  ifstream config_file;
+  config_file.open(filename, std::ifstream::in);
+
+  string line;
+  string key;
+  string value;
+  while (getline(config_file, line)) {
+    // Skip empty lines.
+    if (line.empty()) {
+      continue;
+    }
+
+    // Parse each line.
+    istringstream is_line(line);
```

Comment:
`iss_line` maybe? (input string stream line)

Reply:
As you wish.

Reply:
Fixed in ae5ea3e3acb7a2f5c0fe74755608110cb96c3936.